components/generic_modal.tsx

```diff
@@ -23,6 +23,7 @@ type Props = {
     autoCloseOnCancelButton?: boolean;
     autoCloseOnConfirmButton?: boolean;
     enforceFocus?: boolean;
+    container?: any;
```

Comment:
why `any`? cant we be a bit more strict here?

Reply:
I was following the type convention for the react bootstrap Modal for consistency :D but we can restrict it to a node for sure.